pom.xml

```diff
@@ -64,6 +64,7 @@
     </scm>
     <properties>
         <janusgraph.compatible.versions />
```

Comment:
hi @mbrukman, I wondered the same thing, but the following code checks first thing current version..

```java
if (!JanusGraphConstants.VERSION.equals(version) && !JanusGraphConstants.COMPATIBLE_VERSIONS.contains(version)) {
```

but just in case, I brought up a brand new empty cassandra install and ran quite a few tests, works just fine.